webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -198,52 +193,35 @@ class ManageManifestModal extends Component {
 
                     <FormGroup>
                       <Grid>
-                        <h5>{__('Subscription Manifest')}</h5>
+                        <h3>{__('Subscription Manifest')}</h3>
                         <hr />
                         <Row>
-                          <Col sm={4}>
-                            <ControlLabel
-                              className="control-label"
-                              htmlFor="usmaFile"
-                              style={{ paddingTop: '0' }}
-                            >
-                              <OverlayTrigger
-                                overlay={
-                                  <Tooltip id="usma-tooltip">
-                                    {__('Upstream Subscription Management Application')}
-                                  </Tooltip>
-                                }
-                                placement="bottom"
-                                trigger={['hover', 'focus']}
-                                rootClose={false}
-                              >
-                                <div>{__('USMA')}</div>
-                              </OverlayTrigger>
-                            </ControlLabel>
+                          <SimpleContentAccess
+                            enableSimpleContentAccess={enableSimpleContentAccess}
+                            disableSimpleContentAccess={disableSimpleContentAccess}
+                            isSimpleContentAccessEnabled={simpleContentAccess}
+                            canToggleSimpleContentAccess={!disableSCASwitch}
+                          />
+                        </Row>
+                        <Row>
+                          <Col sm={5}>
+                            <strong>{__('Subscription Allocation')}</strong>
                           </Col>
-                          <Col sm={8}>
+                          <Col sm={7}>
                             {getManifestName()}
                           </Col>
-                        </Row>
-                        <Row>
-                          <Col sm={4}>
-                            <div>{__('Simple Content Access')}</div>
+                          <Col sm={5}>
```

Comment:
```suggestion
                        </Row>
                        <Row>
                          <Col sm={5}>
```
Not strictly required, but this will prevent Subscription Allocation and Import New Manifest from being in the same row